eo-maven-plugin/src/main/java/org/eolang/maven/DcsNoOneHasTransitive.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import org.apache.maven.model.Dependency;
+import org.cactoos.Func;
+import org.cactoos.list.ListOf;
+import org.cactoos.scalar.LengthOf;
+
+/**
+ * Check if all dependencies have transitive dependencies.
+
+ * @since 0.28.11
+ * @todo #1361:90min The class without tests. We definitely have to write some tests
+ *   in order to make the class possible for future refactoring and easy maintainance.
+ */
+final class DcsNoOneHasTransitive implements Dependencies {
+
+    /**
+     * Ignore that filter if flag is true.
+     */
+    private final boolean ignore;
+
+    /**
+     * Source of dependencies to check.
+     */
+    private final Dependencies delegate;
+
+    /**
+     * The function that get all transitive dependencies for the particular one.
+     */
+    private final Func<Dependency, Dependencies> transitive;
+
+    /**
+     * The main constructor.
+     *
+     * @param ignore Ignore that filter if flag is true.
+     * @param delegate Source of dependencies to check
+     * @param transitive The function that get all transitive dependencies for the particular one.
+     */
+    DcsNoOneHasTransitive(
+        final boolean ignore,
+        final Dependencies delegate,
+        final Func<Dependency, Dependencies> transitive
+    ) {
+        this.ignore = ignore;
```

Comment:
@volodya-lombrozo read this, please: https://www.yegor256.com/2016/04/19/object-must-not-be-configurable.html

Reply:
@yegor256 fixed.